README.md

```diff
@@ -108,14 +121,15 @@ to add  ```"50505:50505"``` to the northbound ```ports``` block in ```docker-com
 northbound:
   container_name: openkilda-northbound
   build:
-    context: services/src/northbound/
+    context: docker
+    dockerfile: northbound/Dockerfile
   ports:
     - '8088:8080'
     - '50505:50505'
```

Comment:
nit: don't mind to align this port with one from https://github.com/telstra/open-kilda/blob/develop/docker/floodlight-modules/run.sh#L19